src/main/java/core/basesyntax/servise/FileWriteService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.servise;
+
+import java.util.Map;
+
+public interface FileWriteService {
+    void writeToFile(Map<String, Integer> fruitStorage);
```

Comment:
let's pass here not `Map<String, Integer>` but `List<String>`, for example (products which are already parsed to a string form)

in your StorageService you can have `getAllRemainingFruit()` method